### PR TITLE
adds anyhow support

### PR DESCRIPTION
**Pull request by lattice0**
Adds optional support for implementing `IntoDart` for `anyhow`, needed on https://github.com/fzyzcjy/flutter_rust_bridge/issues/533

### CODE REVIEW

File: src/into_dart.rs [hunk 1]
```diff
@@ -43,6 +45,13 @@ impl IntoDart for () {
     }
 }
 
+#[cfg(feature="anyhow")]
+impl IntoDart for anyhow::Error {
+    fn into_dart(self) -> DartCObject {
+        self.to_string().into_dart()
```
Comment (fzyzcjy) [inline src/into_dart.rs:51]: Hmm I am wondering whether this is sufficient or not? By https://docs.rs/anyhow/1.0.58/anyhow/struct.Error.html, seems that the debug format `{:?}` prints out more information?

Or, if we want more structured solutions we can format it by ourselves

**Comment by fzyzcjy** [comment]
This is blocking downstream library flutter_rust_bridge. May I know when will it be merged? Thanks

/cc @jerel @shekohex (not sure who is maintaining this lib)

File: Cargo.toml [hunk 2]
```diff
@@ -17,6 +17,7 @@ license-file = "LICENSE"
 [dependencies]
 pin-project = { version = "1.0.8", optional = true }
 atomic = "0.5"
+anyhow = {version="1.0.58", optional = true}
```
Comment (shekohex) [inline Cargo.toml:20]:
```suggestion
anyhow = { version = "1.0.58", optional = true }
```

**Review by shekohex — CHANGES_REQUESTED** [review]
Just a few comments, linting and formatting, and it should be ready to be merged.

**Comment by shekohex** [comment]
any updates on this @lattice0 and @fzyzcjy ?

**Comment by lattice0** [comment]
Will we go with a simple string @fzyzcjy?

**Comment by fzyzcjy** [comment]
@lattice0 As suggested above

> Hmm I am wondering whether this is sufficient or not? By https://docs.rs/anyhow/1.0.58/anyhow/struct.Error.html, seems that the debug format {:?} prints out more information?
Or, if we want more structured solutions we can format it by ourselves

At least maybe use `{:?}` so we have more info

**Comment by shekohex** [comment]
Try `cargo +nightly format` them push your changes.

**Comment by lattice0** [comment]
`cargo +nightly fmt`

changed 6 files, is it ok to push?

**Comment by shekohex** [comment]
Yup!

**Review by shekohex — APPROVED** [review]

**Comment by shekohex** [comment]
Thank you, will publish a new version very soon!

**Comment by shekohex** [comment]
Released in `v0.1.13`.

**Pull request merged**